src/main/kotlin/org/ozwillo/dcimporter/web/DataAccessRequestHandler.kt

```diff
@@ -79,6 +82,16 @@ class DataAccessRequestHandler(
                                 fields = currentDataAccessRequest.fields
                             )
                         ).subscribe()
+
+                        try {
```

Comment:
just add a TODO (or file an issue) for a later decoupling of the mail sending feature (but ok for now)